### PR TITLE
Profile

### PR DESCRIPTION
**Pull request by josiahtillman**

### CODE REVIEW

File: app/static/src/css/custom.scss [hunk 1]
```diff
@@ -132,6 +132,56 @@ body {
   text-decoration: line-through;
 }
 
+.circular-image {
```
Comment (c-schwarze) [inline app/static/src/css/custom.scss]: Should these edits go in the profile.css file?

File: app/templates/base.html [hunk 2]
```diff
@@ -65,9 +65,11 @@ <h1 class="my-4">{% block tab_title %}{{ tab_title|title }}{% endblock %}</h1>
 
             {% if request.path == '/checklist' %}
                 {% include 'modules/checklist.html' %}
-            {% endif %}
 
-            {% if request.path != '/checklist' %}
+            {% elif request.path == '/profile' %}
```
Comment (c-schwarze) [inline app/templates/base.html]: This seems not needed. We should be able to create the route fully in python.

File: app/templates/modules/navbar.html [hunk 3]
```diff
@@ -13,7 +13,7 @@
             <span class="hidden-sm-down">{{ session['user_profile']['pref_first_last_name'] }}</span>
         </a>
         <div class="dropdown-menu  dropdown-menu-right">
-            <a class="dropdown-item" href="/deep_link_channel/cf0a32d58c5865135ae8b6ac2d2f1ce8/home">Profile</a>
+            <a class="dropdown-item" href="/profile">Profile</a>
```
Comment (c-schwarze) [inline app/templates/modules/navbar.html]: this should use url_for

File: app/templates/modules/profile.html [hunk 4]
```diff
@@ -0,0 +1,77 @@
+{% macro profile_item(display1, key) -%}
```
Comment (c-schwarze) [inline app/templates/modules/profile.html]: I like what you did here 👍

File: app/templates/base.html (hunk 2 above)
Comment (josiahtillman) [inline app/templates/base.html]: Yea, I talked to eric about this, and he said we would refactor this later as this was a temporary fix for the checklist

Comment (c-schwarze) [inline app/templates/base.html]: 👍

**Pull request merged**